codes/quantum/qubits/quantum_cyclic.yml

```diff
@@ -0,0 +1,117 @@
+features:
+
+  # The fields below relate to specific properties of the code that are worth
+  # noting. In some cases (see below), there might be multiple pieces of
+  # information associated with a field (e.g., mutliple encoders/decoders,
+  # multiple facts about fault tolerance, etc.); in such cases, the field
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: 'Unknown.'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  # encoders: 
+  #   - 'Unitary circuit of depth ... \cite{arxiv:old-paper}.'
+  #   - 'Measurement-based preparation ... with ancilla overhead of ... .'
+  #   - 'Leave discussion of fault tolerance to fault-tolerance field.'
+
+  # transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  # general_gates:
+  #   - 'Universal gate set achieved by either additional ... gate.'
+  #   - 'Magic-state distillation protocols'
+  #   - 'kth Clifford hierarchy gates obtained by ... circuits'
+
+  decoders: 
+    - 'Adapted from the Berlekamp decoding algorithm for classical BCH codes \cite{arXiv:1007.1697}.'
+    # - 'Just-in-time decoder with ... \cite{arxiv:awesome-paper}.'
+
+  # fault_tolerance:
+  #   - 'Transversal gates are fault-tolerant w.r.t. ... noise \cite{doi:ok-paper}'
+  #   - 'Other fault-tolerant gadgets (measurements, encoders, error correcting steps)'
+  #   - 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+  #   - 'Pieceable fault tolerance.'
+
+  threshold: 
+    - 'Unknown'
+  
+  # Propose a field that you feel is applicable to a large and interesting class of codes
+  #some_other_field:
+  # - 'detail'
+
+# realizations:
+#   # List and explain the different "domains" of realizations in list items.
+#   # Group closely related realizations into a single item. It's OK to have a
+#   # single list item. Each item should more or less correspond to what would be
+#   # said in a paragraph.
+#   - 'Not directly implemented, but BCH codes form a subclass, and are used in DVD, solid state drive storage, etc.'
+
+notes:
+  - '\cite{arXiv:1011.5814} gives a similar construction for quantum cyclic codes.'
+  - 'Kitaev [[5,1,3]] code is an example.'
+  - 'All examples have been found by computer algebra programs. \cite{arXiv:1007.1697} give examples of [[17,1,7]] and [[17,9,3]] quantum cyclic codes.'
+  # - 'Bounds on n,k, or d for this class.'
+  # - 'Links to code tables, github, GAP algebra packages, more papers \cite{arXiv:####.#####}.'
+  # - 'Connections to physics; based on \(H^4(\mathcal{BG},\mathbb{Z})\)'
+  # - 'Can reference other codes, but try to put such connections in the relations fields.'
+  # - 'Anything noteworthy that''s not above or below.'
+
+relations:
+  parents:
+    - code_id: stabilizer
```

Comment:
The paper abstract says their defn is more general than stabilizer codes. If so, use qubits_into_qubits.